osl-incubator/sugar
# Change the default arg parser to typer

## Pull Request description
Change the default arg parser to typer



## How to test these changes

* Run smoke and unit tests to see if there's no errors
* use the new cli sugar --help

* ```...```


## Pull Request checklists

This PR is a:

About this PR:

Author's checklist:

## Additional information





## Reviewer's checklist

Copy and paste this template for your review's note:

```
## Reviewer's Checklist

```
